source/install/ee-install.rst

```diff
@@ -41,6 +41,17 @@ Upgrading to Enterprise Edition from Mattermost Team Edition
 
 If you need to migrate Team Edition prior to install, `please follow the migration guide. <http://docs.mattermost.com/administration/migrating.html>`_
 
+Upgrading to Enterprise Edition in GitLab Omnibus
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GitLab Omnibus runs the open source Mattermost Team Edition. To upgrade to Mattermost Enterprise Edition, follow these steps:
+
+1) Disable the built-in Mattermost instance on // XXX Harrison: Where do they go disable it?
```

Comment:
Harrison: If you can help with this step

Reply:
To disable it, they go open `/etc/gitlab/gitlab.rb` and set the following line to false
```ruby
mattermost['enable'] = false
```
And then they run `sudo gitlab-ctl reconfigure` to apply the updated settings